src/HypercertExchangeClient.ts

```diff
@@ -209,7 +201,7 @@ export class HypercertExchangeClient {
       endTime: endTime,
       price: price,
       itemIds: itemIds,
-      amounts: amounts,
+      amounts: itemIds.map(_ => 1n),
```

Comment:
Resolved the conversation, but this is an update in business logic. So create a unit test, release an alpha and validate on testnet.

Reply:
Tested and works in https://github.com/hypercerts-org/hypercerts-app/pull/428.

Reply:
Wait they're completely broken? Let's create an issue for that, we should be able to get green tests and automate package updates from our CI

Reply:
Yes, they have been almost since the beginning, and we've spent some time trying to get it to work as well but that was forever ago. Biggest issue is hardhat setup within the tests.

Reply:
Created an issue here: https://github.com/hypercerts-org/marketplace-sdk/issues/30